internal/kafka/internal/presenters/data_plane_cluster_status.go

```diff
@@ -21,9 +21,18 @@ func ConvertDataPlaneClusterStatus(status private.DataPlaneClusterUpdateStatusRe
 		})
 	}
 
+	dynamicCapacityInfo := map[string]api.DynamicCapacityInfo{}
```

Comment:
if this check required here?

``` d.DataplaneClusterConfig.IsDataPlaneAutoScalingEnabled() ``` ?

Reply:
The check is already done in the service layer.